streaming/index.js

```diff
@@ -145,8 +147,13 @@ const startServer = async () => {
   });
 
   const app = express();
+  const trustProxy = proxyaddr.compile(
+    process.env.TRUSTED_PROXY_IP ?
+      process.env.TRUSTED_PROXY_IP.split(/(?:\s*,\s*|\s+)/) :
+      ['loopback', 'uniquelocal']
+  );
 
-  app.set('trust proxy', process.env.TRUSTED_PROXY_IP ? process.env.TRUSTED_PROXY_IP.split(/(?:\s*,\s*|\s+)/) : 'loopback,uniquelocal');
```

Comment:
This default of a string of multiple values actually caused proxyaddr to throw a server crashing error if the req.ip property was ever accessed